tests/integration/hive_metastore/test_migrate.py

```diff
@@ -460,6 +460,7 @@ def prepared_principal_acl(runtime_ctx, env_or_skip, make_dbfs_data_copy, make_c
     ]
     runtime_ctx.with_table_mapping_rules(rules)
     runtime_ctx.with_dummy_azure_resource_permission()
+    runtime_ctx.with_dummy_aws_resource_permission()
```

Comment:
```suggestion
```
Test has to pass without these dummy locations, also remove these for azure. It looks like a substantial bug in the logic if we require this in every test.